debian/control

```diff
@@ -33,39 +41,17 @@ Description: Go programming language compiler
  fast, statically typed, compiled language that feels like a
  dynamically typed, interpreted language.
  .
- This package provides an assembler, a compiler, and a linker for the
- Go programming language. This is Google's implementation of the Go
- tool chain.
-
-Package: golang-src
-Replaces: golang-weekly-src, golang-go (<< 2:1-3~)
-Conflicts: golang-weekly-src
-Provides: golang-weekly-src
-Breaks: golang-go (<< 2:1-3~)
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: Go programming language compiler - source files
- The Go programming language is an open source project to make
- programmers more productive. Go is expressive, concise, clean, and
- efficient. Its concurrency mechanisms make it easy to write programs
- that get the most out of multicore and networked machines, while its
- novel type system enables flexible and modular program construction.
- Go compiles quickly to machine code yet has the convenience of
- garbage collection and the power of run-time reflection. It's a
- fast, statically typed, compiled language that feels like a
- dynamically typed, interpreted language.
+ This package provides an assembler, compiler, linker, source, and compiled
+ libraries for the Go programming language.
  .
- This package provides the Go programming language source files needed
- for godoc and goinstall.
+ Go supports cross-compilation, but as of Go 1.5, it is no longer necessary to
+ pre-compile the standard library inside GOROOT for cross-compilation to work.
```

Comment:
See http://dave.cheney.net/2015/03/03/cross-compilation-just-got-a-whole-lot-better-in-go-1-5
for a decent explanation from Dave about the update. :+1: